SIL.DblBundle/DblMetadata.cs

```diff
@@ -55,15 +55,29 @@ public class DblMetadata
 	public abstract class DblMetadataBase<TL> : DblMetadata, IProjectInfo where TL: DblMetadataLanguage, new()
 	{
 		/// <summary>
-		/// Loads information about a Digital Bible Library bundle from the specified projectFilePath.
+		/// Loads information about a Digital Bible Library bundle from the specified reader.
 		/// </summary>
-		public static T Load<T>(string projectFilePath, out Exception exception) where T : DblMetadataBase<TL>
+		/// <param name="metadataReader">A TextReader object assumed to be positioned at the start of the </param>
+		/// <param name="resourceIdentifier">Filename or other meaningful identifier that can be used in an error message if there is a
```

Comment:
Can you add line breaks so that we have around 100 columns max? That would make it easier to read when looking at the code on GitHub without having to scroll horizontally.